lib/ldap_fluff/generic_member_service.rb

```diff
@@ -70,7 +70,7 @@ def get_logins(userlist)
 
   def get_login_from_entry(entry)
     [@attr_login, 'uid', 'cn'].each do |attribute|
-      return entry.send(attribute) if entry.respond_to? attribute
+      return entry.send(attribute.map(&:downcase)) if entry.respond_to? attribute
```

Comment:
Yeah, the one in generic_member_service should not be there I don't think.  We shouldn't add to the problem.

Foreman stores usernames preserving their case, but matches them case insensitively - so it should handle having a lower case username stored ("domcleal") and uppercase in LDAP or elsewhere ("DomCleal").  It should also handle the reverse - if we downcase here, we only fix the case where a Foreman username is lower case.  It might break people's existing setups if their Foreman usernames currently match the case of LDAP.